ts/src/index.ts

```diff
@@ -112,27 +108,56 @@ function initConfigLocal(config: Config): ProfilerConfig {
   return mergedConfig;
 }
 
+async function getGcpMetadata(
```

Comment:
"Gcp" or "GCP"? Not sure how the abbreviation are usually spelled in Node.js code, make sure it's consistent across.

Reply:
In Node.js, only the first letter of acronyms are capitalized when in 
 the middle of camel case variable names. 
So, this is consistent with variables names in the agent, and in libraries it depends on.

Reply:
`getGcpMetadata` is correct. The js style guide would apply: go/js-style#naming-camel-case-defined (internal link).